2.train-VAE/scripts/nbconverted/3.beta_vae_training.py

```diff
@@ -149,21 +145,21 @@
 plt.show()
 
 
-# In[14]:
+# In[57]:
```

Comment:
Please make sure that all cells in jupyter notebooks are run sequentially. This is to ensure that every variable is changed in expected ways (no hidden cell executions to change states unknowingly!)

Reply:
Yes, thanks! Did I make that error in this spot?

Reply:
I believe so. the number inside brackets (e.g. `# In[14]`) implies execution order. i.e. originally this cell was executed in place 14 whereas the change your proposing is executed in place 57.

the PR doesn't have to revert back to 14 if it doesn't make sense, but the first cell needs to start with 1 and every cell needs to be sequentially ordered with no spacing.

Reply:
Ah! Got it. That must've happened because I was reproducing that plot many times after changing the hyperparameter values. Good catch, I'll fix it!